app/services/twilio/twilio_text_messenger.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Twilio
+  class TwilioTextMessenger < ApplicationService
+    attr_reader :message
+
+    def initialize(message)
+      @message = message
+    end
+
+    def call
+      client = Twilio::REST::Client.new
+      client.messages.create({
+        from: ENV['twilio_phone_number'],
+        to: '+111111111111',
+        body: message
+      })
+    end
+  end
+end
```

Comment:
please check examples how all services are organized

Reply:
done

Reply:
used `Twilio::REST::TwilioError` like in Twilio docs